code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1046,6 +1046,51 @@
 	..()
 	. = 1
 
+/datum/reagent/medicine/clotagen
+	name = "Clotagen"
+	description = "Encourages coagulation to stop wounds from bleeding."
+	color = "#FA7DB5"
+	chem_flags = CHEMICAL_RNG_GENERAL | CHEMICAL_RNG_FUN | CHEMICAL_RNG_BOTANY
+	taste_description = "a roll of gauze"
+	metabolization_rate = 0.3 * REAGENTS_METABOLISM
+	overdose_threshold = 20
+
+/datum/reagent/medicine/clotagen/on_mob_life(mob/living/carbon/M)
+	M.adjustBruteLoss(-0.3)
+	M.suppress_bloodloss(0.3)
+	..()
+
+/datum/reagent/medicine/clotagen/overdose_process(mob/living/M)
+	M.adjustBruteLoss(1)
+	M.adjustOrganLoss(ORGAN_SLOT_HEART, 1)
+	..()
+
+/datum/reagent/medicine/clotagenp
+	name = "Clotagen Plus"
+	description = "Encourages coagulation to stop wounds from bleeding, as a side effect it also increases blood toxicity."
+	color = "#82405D"
+	chem_flags = CHEMICAL_RNG_GENERAL | CHEMICAL_RNG_FUN | CHEMICAL_RNG_BOTANY
+	taste_description = "a roll of gauze with a hint of toxin"
+	metabolization_rate = 1 * REAGENTS_METABOLISM
+	overdose_threshold = 10
+
+/datum/reagent/medicine/clotagenp/on_mob_life(mob/living/carbon/M)
+	M.suppress_bloodloss(0.7)
```

Comment:
Might as well just make this 0.3 too, since that alone is extremely powerful (you wouldnt expect someone to have more than 3 bleeding)

Reply:
Then what would be the purpose of clotagen plus if its almost exactly the same as normal clotagen but you gain toxin damage?

Reply:
It grants you complete immunity to bleeding, which should be incredibly strong since it stops all bleeding. Arguably, the bleed suppression effect of both should be weak so that its not overshadowed by the otherone being able to heal all bleed damage in a few seconds.

Reply:
> It grants you complete immunity to bleeding, which should be incredibly strong since it stops all bleeding. Arguably, the bleed suppression effect of both should be weak so that its not overshadowed by the otherone being able to heal all bleed damage in a few seconds.

Good point yeah, I'll add this now